okhttp/src/main/java/com/squareup/okhttp/TlsVersion.java

```diff
@@ -43,4 +43,12 @@ public static TlsVersion forJavaName(String javaName) {
     }
     throw new IllegalArgumentException("Unexpected TLS version: " + javaName);
   }
+
+  public static String[] javaNames(TlsVersion... tlsVersions) {
```

Comment:
Need this be public? Maybe adding a single accessor javaName() would be sufficient for what's public?